packages/app/src/runs/modals/CreateCloudOrgModal.vue

```diff
@@ -101,28 +94,19 @@ const props = defineProps<{
 }>()
 
 const query = useQuery({
-  query: CheckCloudOrganizationsDocument,
+  query: CloudOrganizationsCheckDocument,
   requestPolicy: 'network-only',
+  pause: true,
 })
 
-const polling = ref(false)
+const refetch = useDebounceFn(() => query.executeQuery(), 1000)
 
-const { pause, resume } = useIntervalFn(() => {
-  if (props?.gql?.organizationControl?.nodes?.length || 0 > 0) {
-    pause()
-  } else {
-    query.executeQuery()
-  }
-}, 5000)
+const polling = ref(false)
```

Comment:
Is `polling` still the best name for this? Should is be `waitingForOrg`, `startWaitingForOrg()` or something like that?